Nitrox.Assets.Subnautica/LanguageFiles/English.json

```diff
@@ -38,7 +38,7 @@
     "Nitrox_WaitingPassword": "Waiting for server password input…",
     "Nitrox_WaitingUserInput": "Waiting for user input…",
     "Nitrox_LaunchGame": "Launching game…",
-    "Nitrox_SilencedChatNotif": "Chat is now muted",
```

Comment:
I would like to move all language file changes to a separate pr as I fear a merge conflict with weblate. Therefor this new pr will also be merged after the weblate pr, meaning shortly before the next release.